galera/src/replicator_smm.cpp

```diff
@@ -3150,6 +3150,7 @@ void galera::ReplicatorSMM::process_prim_conf_change(void* recv_ctx,
     /* Invalidate sst_seqno_ in case of group change. */
     if (state_uuid_ != group_uuid) sst_seqno_ = WSREP_SEQNO_UNDEFINED;
 
+    bool mixed_ver_cluster = view_info->proto_ver < 4;
```

Comment:
Use `const` similar to all local var initializations in this function.
```suggestion
    bool const mixed_ver_cluster = view_info->proto_ver < 4;
```

Reply:
done